template/web/app/paymaster-bundler/constants.ts

```diff
@@ -0,0 +1,28 @@
+export const rpcUrl = process.env.NEXT_PUBLIC_RPC_URL;
+export const paymasterUrl = process.env.NEXT_PUBLIC_PAYMASTER_URL;
+export const privyID = 'clsxnba7r00tragoglekx9or7';
```

Comment:
This safe to checkin?

Reply:
No, removed and @0xAlec will cycle keys.